### DISCUSSION TITLE
Discussion on Migrating Pulsar-Specific Docs to `pulsar-edit.github.io` from `.github`

### DISCUSSION THREAD

**Discussion by ghost** [pulsar-edit/.github#33] [General]
This is a discussion thread on migrating Pulsar-specific docs to the website (`pulsar-edit.github.io`) from .github.

**Comment by confused-Techie** [comment]
Yeah I'm all for this. Some things will need to remain there to properly link to other locations.

But I would be willing to have a shorter CONTRIBUTING.md and instead have a link to the website that goes much more in-depth.

Then especially other things like the repo collection and what not could very easily be moved over.

But yeah totally on board with this idea.

**Reply by kaosine** [reply]
This is definitely the more ideal solution to go after. I'd put all these in the section's folder under the FAQ that @Daeraxa made the other day. That way we can import them, and if need be reuse them as seen fit. We're trying to get to implementing aliases to various content over there, and these would be something we need to do on the site if these move there. Since we could use them on other pages as seen fit, and include a link to the full faq under it.

**Comment by meadowsys** [comment]
we _could_ keep them in `.github` and then write some scripts to download them, so they can be used by the site? I kind of think this is a higher maintenance and janky solution though?

**Reply by kaosine** [reply]
There's that, but man it's weird how working with it without "init'ing" it as a package which idk

**Comment by Daeraxa** [comment]
To me it makes sense that org wide specific things should stay in `.github`. Code of conduct, contributing, templates etc. etc. but anything that doesn't need to work org wide should be moved to the website repo.
(Within reason of course, no need for going out of the way to do anything if it makes life harder)
